# initial_range: [1.0, 1.0, 1.0]
initial_range: [0.05, 0.05, 0.05]
input_std: [0.005, 0.005, 0.005, 0.005]
meas_std_val: 0.05
num_particles: 50
